tox.ini

```diff
@@ -11,6 +11,7 @@ passenv =
     HOME
     NO_TESTS_OVER_WIRE
     NO_CREDENTIALS_TESTS
+    SCRIPTWORKER_GITHUB_TOKEN
```

Comment:
I confirm we do need this env var passed down per:

https://github.com/mozilla-releng/scriptworker/blob/98b1fe9ffea85bb42a3cd29b0f1a1e20511914f5/tests/test_production.py#L59-L60